Police-Data-Accessibility-Project/data-sources-app-v2
# refactor: update client auth per new API strategy

## To test

Run locally, ensure that you can:

1. Visit `/sign-up` to create a new user.
2. Visit `/sign-out` to log them out
3. Visit `/sign-in` to log them back in again
